.github/workflows/pre-release.yaml

```diff
@@ -61,8 +61,8 @@ jobs:
       - uses: tibdex/github-app-token@v1
         id: generate-token
         with:
-          app_id: ${{ secrets.APP_ID }}
-          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+          app_id: ${{ secrets.ODH_RELEASE_APP_ID }}
```

Comment:
are these 2 secrets maintained by DevOps ?

Reply:
yes that is correct, only to get the token. I guess we could move this to repo-level secret. @VaishnaviHire wdyt?

Reply:
ok, thats fine. as long as we (someone in the team) has permission to maintain it.
i was worried that we need to rely on DevOps if something urgent